code-challenges.test.js

```diff
@@ -23,12 +31,38 @@ const hitchhikersCharacters = [
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// Pseudo code:  Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
+
+// Declare function using const
+// Pass in nameCapped as the parameter
+// iterate over array using .map to get values in object
+// Destrcture object (use curly braces) to get the name and occupation (new variables)
+//Use char at to capitilaze name
+
+hitchhikersCharacters.map(nameCapped => {
+  const {name, occupation} = nameCapped
+  console.log (`${name} is a ${occupation} `)
+})
+ //(`${name} is a ${occupation} `) <--- I know a console log is not needed with Jest but I tried outside of the scope to console log it and there was an error, so I kept it like so.
+
+//tried to put chartAt in different places but was unable to figure out where in the function it would go.
```

Comment:
The issue here is that you are not declaring a function. You are using the higher order function on the provided variable. The error is probably from your test trying to find the function `nameCapped` in order to run it and test it, but not finding it anywhere since there is no function declared, which will cause a ReferenceError.